### PR TITLE
Pass marked files to rifle

### PR DESCRIPTION
**Pull request by Gallipo**
If there are files marked, pass a list of those to rifle (instead of the file under 'point'). If there are no marked files, pass the file under 'point' (which is the behaviour before the change).

### CODE REVIEW

**Comment by Gallipo** [comment]
Obviously, this would benefit from an option to switch the behaviour ('marked files' vs. 'file under point'). Unfortunately C-u and C-u C-u are already in use.

**Comment by vifon** [comment]
I've considered adding a similar feature but it's not obvious how it would fit into the UI. Personally I consider the ability to operate on the current file while marking files one by one more important, so I'm not going to merge it in the current form, but I'm totally open to suggestions how to integrate it better into the UI.

I'm aware the prefix argument is already very crowded here. Crowded enough that I don't even use it most of the time as it's too confusing, so if you'd like to rework this part, definitely go for it.

Bottom line: I like this feature but the current UI isn't ready for it.
